src/views/DashboardView.vue

```diff
@@ -136,12 +126,12 @@ export default {
 
                 <div id="boxes-wrap" class="flex justify-between caveat text-3xl mx-auto">
                         <!-- BOXES: INCOMPLETE BY PRIORITY -->
```

Comment:
tweak: conditional format should not depend on there being tasks on list (change it to depend on order/index?)